next.config.js

```diff
@@ -2,7 +2,10 @@ module.exports = {
   reactStrictMode: true,
 
   images: {
-    domains: ['127.0.0.1', 'placekitten.com'],
+    domains: [
+      'placekitten.com',
+      'community-fridge-map-images-prod.s3.amazonaws.com',
```

Comment:
same comment as before, adding for dev: community-fridge-map-images-dev.s3.amazonaws.com